compiler/src/main/kotlin/io/github/apl_cornell/viaduct/security/solver2/Terms.kt

```diff
@@ -68,3 +68,31 @@
             )
         }
     )
+
+fun <C : HeytingAlgebra<C>, V, T> Term<C, V>.integrityFlowsTo(
```

Comment:
The above function (`.flowsTo`) should call `integrityFlowsTo` and `confidentialityFlowsTo` instead of duplicating code.